src/easier68k/assembler/assembler.py

```diff
@@ -0,0 +1,48 @@
+from ..core.util.parsing import strip_comments, has_label, get_label, strip_label, get_opcode, strip_opcode, parse_literal
+import binascii
+import sys
+from ..core import opcodes
+from ..core.opcodes import *
+
+
+def parse(text: str):  # should return a ListFile and errors/warnings eventually
```

Comment:
can we include docstring that explains what this method does?

Reply:
Yup, this is still a WIP, I plan on filling things out much better tonight.